### PR TITLE
Resolve several bugs

### PR DESCRIPTION
**Pull request by PoundsOfFlesh**
**Overall Review of Changes:**
I discovered several issues while testing on RHEL 9.3. The commits describe what I fixed. 

**Issue Fixes:**
N/A

**Enhancements:**
N/A

**How has this been tested?:**
I tested by installing a RHEL 9.3 virtual machine and running the playbook.

### CODE REVIEW

**Comment by fallenpixel** [comment]
I'm not affiliated with the project at all. I'd suggest you take a look at the [contributing guide](https://github.com/ansible-lockdown/RHEL9-STIG/blob/main/CONTRIBUTING.md) included in the repository. It discusses the requirements for the repository. Most importantly, your commits must be signed and must have a git signoff. This is likely not going to go anywhere until those requirements are met.

**Comment by uk-bolly** [comment]
hi @PoundsOfFlesh 

Thank you for taking find, feedback issues and raise the subsequent PR. Unfortunely a couple of issues have arisen.
We are missing the both the sign-off-by and the gog signing of every commit in the PR.
It was also trying to push directly into the main branch. I have changed this to devel but this is not allowing me to rerun with the new setting.
Could i possibly ask if you could cancel this PR and address the issues listed so we can get this pulled?

many thanks

uk-bolly

File: tasks/Cat2/RHEL-09-27xxxx.yml
Comment (uk-bolly) [inline tasks/Cat2/RHEL-09-27xxxx.yml:1]: line 342 is the pink background file always there?

**Comment by PoundsOfFlesh** [comment]
I'm canceling this PR as requested until I figure out how to submit one properly.

**Pull request closed without merging**